### DISCUSSION TITLE
Add support gRPC as in jetbrains HTTP client

### DISCUSSION THREAD

**Discussion by micronull** [Orange-OpenSource/hurl#1097] [Ideas]
https://www.jetbrains.com/help/idea/http-client-in-product-code-editor.html#grpc-requests

The http client jb supports grpc requests. I wish hurl would support them too.

**Comment by fabricereix** [comment]
Hi @micronull 
That will be indeed good to support it. Unfortunately, this is it not something in our priority now.

**Reply by kingkero** [reply]
Do you think it would require a lot of work? There are libraries like [gRPCurl](https://github.com/fullstorydev/grpcurl) which allow sending grpc requests from the CLI quite easily.

I'm assuming an implementation could be similarly done as https://github.com/Orange-OpenSource/hurl/blob/master/packages/hurl/src/http/curl_cmd.rs - is that correct? If not, could you point in the correct direction?
